src/kernels/detect_llvm_amdgcn_buffer_atomic_fadd_f32_float.cpp

```diff
@@ -0,0 +1,21 @@
+#ifndef MIOPEN_DONT_USE_HIP_RUNTIME_HEADERS
+#include "hip/hip_runtime.h"
+#endif
+
+typedef int32_t int32x4_t __attribute__((ext_vector_type(4)));
+
+__device__ float
+__llvm_amdgcn_buffer_atomic_add_f32(float vdata,
```

Comment:
This is ___global___ function definition that has ___reserved___ identifier from both C++ Standard (begins with `__`) and llvm (begins with `__llvm` ) points of view. This means UB.

Reply:
Please correct me if I am wrong here.

Reply:
I'm not sure. I will remove ```__``` in future PR.